### ISSUE TITLE
Architecture definition needed

### ISSUE THREAD

**Issue by arjamizo** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#213]
We definitely need to specify architecture in e.g. JavaArch, so we can easily add new features. 

Keywords ADL ETL ESTL CoSy Unicon ports connnectors.

**Comment by ckuhn203** [comment]
Yes! By God! How will we know what our architecture is if we don't have a 200 page document describing it!? I'll requisition a team to get on this right away!
